table_engine/src/provider.rs

```diff
@@ -72,14 +75,21 @@ impl TableProviderAdapter {
         limit: Option<usize>,
         read_order: ReadOrder,
     ) -> Result<Arc<dyn ExecutionPlan>> {
+        let request_id = RequestId::from(state.config.config_options().get_u64(CERESDB_REQUEST_ID));
```

Comment:
We can only set user-defined variables via SessionConfig, since `TableProvider` is returned from a trait defined in DataFusion.